pkg/networkmanager/network_manager.go

```diff
@@ -471,7 +470,7 @@ func (am *NetworkManager) generateNetworkNeighborsEntries(namespace string, netw
 			if am.dnsResolverClient != nil {
 				domain, ok := am.dnsResolverClient.ResolveIPAddress(networkEvent.Destination.IPAddress)
 				if ok {
-					neighborEntry.DNS = domain
+					neighborEntry.DNS = strings.TrimSuffix(domain, ".")
```

Comment:
why do we have to remove it, I don't understand

Reply:
This will also cause us in each event to omit it before comparing to nn